camel-core/src/main/java/org/apache/camel/impl/DefaultCamelContext.java

```diff
@@ -276,6 +280,8 @@
     private final StopWatch stopWatch = new StopWatch(false);
     private Date startDate;
     private ModelJAXBContextFactory modelJAXBContextFactory;
+    private List<TransformerDefinition> transformers = new ArrayList<TransformerDefinition>();
+    private Map<TransformerKey, Transformer> transformerRegistry = new HashMap<TransformerKey, Transformer>();
```

Comment:
I think we should create a TransformerRegistry class like we have EndpointRegistry. Then we can also use that to expose JMX information at runtime etc. 

https://github.com/apache/camel/blob/master/camel-core/src/main/java/org/apache/camel/spi/EndpointRegistry.java

https://github.com/apache/camel/blob/master/camel-core/src/main/java/org/apache/camel/management/mbean/ManagedEndpointRegistry.java